packages/core/content-type-builder/admin/src/components/FormModalSubHeader/index.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useIntl } from 'react-intl';
+import { Typography } from '@strapi/design-system/Typography';
+import upperFirst from 'lodash/upperFirst';
+import { getTrad } from '../../utils';
+import getModalTitleSubHeader from './getModalTitleSubHeader';
+
+const FormModalSubHeader = ({
+  actionType,
+  modalType,
+  forTarget,
+  kind,
+  step,
+  attributeType,
+  attributeName,
+  customField,
+}) => {
+  const { formatMessage } = useIntl();
+  const type =
+    modalType === 'customField'
```

Comment:
@soupette is this what you had in mind?

Reply:
I haven't reviewed the code yet but I think you missed the validation part, as you can create a custom field called slug for address which already exists.

Do you want me to take a look at the code first without testing the behavior?

Reply:
Yeah I haven't added the validation yet. I worked on it yesterday and will probably open a new PR for that today. I'll also need to make sure plugins can inject inputs into the forms. If you prefer to wait until I've added all, no problem I can ping you once ready. But if you are able to give this a once over as is that could also be helpful. Whatever works best for you.

Reply:
Here's the validation: https://github.com/strapi/strapi/pull/13935

Reply:
Here's the injected inputs: https://github.com/strapi/strapi/pull/13940